backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -1470,6 +1474,14 @@ trait StandardAsyncExecutionActor
     serviceRegistryActor.putMetadata(jobDescriptor.workflowDescriptor.id, Option(jobDescriptor.key), metadataKeyValues)
   }
 
+  def tellBard(metadataKeyValues: Map[String, Any]): Unit =
+    serviceRegistryActor ! BardEventRequest(
+      TaskSummaryEvent(workflowDescriptor.id,
+                       Option(jobDescriptor.key.propertiesToMap).getOrElse(Map()).asJava,
+                       metadataKeyValues.asJava
+      )
+    )
+
```

Comment:
This seems like the canonical way to get task metadata out of Cromwell, but I'm a little hesitant about the arbitrary key-values. What info is in there? Does it have everything we need? Does it have way more than we need?

Reply:
Is metadata standardized per-cloud? Across clouds?

Reply:
I think your hesitation is warranted. This is one small collection of metadata that backends can choose to create on job termination, it's not standardized across backends and doesn't tell the full story of the job. 

What's the current thinking around requirements here? What information do you need about the job?

Reply:
We just need CPU Hours and Docker Image, but I'd love to include everything you use to generate the Monthly GCP Cromwell Spreadsheet so that we can automate that for you!

Reply:
Cool! While we would love allll the metrics immediately, let's start with Docker image and CPU hours. Can maybe throw in memory hours, since it operates the same way as CPU.

Docker should be easy, you can get it from the runtime attributes in the job descriptor via:
```
RuntimeAttributesValidation.extract(DockerValidation.instance, validatedRuntimeAttributes)
```

For CPU or memory hours, you need both the CPU/memory count for the task and the cloud runtime of the task. The CPU and memory counts are in runtime attributes alongside Docker image, so you can get them as above. Cloud runtime is going to be backend-dependent and I'm not sure we have a representation of it in the GCP backend right now[1]. Katrina is actually adding one for Azure/TES in https://github.com/broadinstitute/cromwell/pull/7415, so you can probably use or iterate on that.

[1] You might ask, then how go we compute core hours for the spreadsheet? We do it by looking at the time between the earliest and latest events that we get from GCP Lifesciences during the task run. We store these events in metadata as a giant bucket of `executionEvents`. Since we're doing this in source code rather than analyzing metadata after the fact, I'm hoping we can do something better.

Reply:
For cloud identity, we have a slightly-used notion of `platform` that we use when interpreting runtime attributes. Currently I think we only *use* it to distinguish between Azure-TES and Not-Azure-TES. You could set the cloud provider in each backend actor implementation, and use that `platform` value to detect when on Azure. We could also choose to fully plumb `platform` through for GCP and depend fully on it.

https://github.com/broadinstitute/cromwell/pull/7380/files